app/views/employee_mailer/pick_up_delivery_instructions_email.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+  </head>
+  <body>
+    <h1>Thank you for purchasing a reward: <%= @reward.title %> !</h1>
+    <p>We kindly inform about receiving order on above reward.</p>
+    <p>Below we leave instructions related to picking up ordered reward.</p>
+    <p>You can pick up your reward from <%= (Time.now + (60*60*24*3)).strftime('%e %B %Y') %></p>
```

Comment:
You can use more informative date format, like `Time.now + 3.days` 
Also, it's recommended to avoid using `Time.now`, and using `Time.current` instead. You can read more here: https://thoughtbot.com/blog/its-about-time-zones

Reply:
First of all thanks for CR :) I have used `Time.current` and `3.days` according to your suggestion 👍 Also thanks for sharing the article, now I know why I should use `.current` ;)